src/src/com/microsoft/aad/adal/PromptBehavior.java

```diff
@@ -48,5 +48,13 @@
      * to passing prompt=refresh_session as an extra query parameter during
      * the authorization.
      */
-    REFRESH_SESSION
+    REFRESH_SESSION, 
+    
+    /**
+     * If Azure Authenticator or Company Portal is installed, this flag will have 
+     * the broker app force the prompt behavior, otherwise it will be same as Always. 
+     * If using embeded flow, please keep using Always, if force_prompt is set for 
+     * embeded flow, the sdk will re-intepret it to Always. 
+     */
+    FORCE_PROMPT
```

Comment:
do you mean in the comment or the enum itself? Enum itself should be just capitalized, i will update the comment to make sure all capitalized.